python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -105,6 +107,19 @@ def optimize(cls, model):
             # todo implement 4.28.0 ~ 4.30.2
             pass
 
+        if "chatglm2" in model.config._name_or_path:
```

Comment:
do we need to check the transformers `version`?

Reply:
There is no `chatglm` model in `transformers`. The model implementation is directly placed into the model repo. I have not find a way to check the version of the model implementation.